src/components/Loader/PageLoader.styles.js

```diff
@@ -2,8 +2,10 @@ import { css } from '@emotion/core';
 
 const styles = {
   root: css({
-    display: 'flex',
-    margin: 'auto',
+    left: '50%',
+    position: 'fixed',
+    top: '50%',
+    transform: 'translate3d(-50%, -50%, 0)',
```

Comment:
these styles need to be adjusted for desktop

Reply:
My bad! Fixed.